src/components/pages/service/service.tsx

```diff
@@ -28,7 +28,11 @@ export const Service = () => {
       transparentHeader={true}
       light={true}
       leadbox={leadbox}
-      hero={<AuroraHero title={t('services.hero')} />}
+      hero={
+        <AuroraHero title={t('services.hero')}>
+          {t('services.hero.description')}{' '}
```

Comment:
we never had a text here and reused the title. You added a text, is that correct?

Reply:
No we didn't but now we have one. I wanted some content on the service page as well. And yes it works fine -> https://satellytescommain21751-updatetext.gtsb.io/de/services/

Reply:
yeah that's good, thanks!